src/ucp/proto/proto_common.h

```diff
@@ -96,6 +96,9 @@ typedef struct {
 
     /* Protocol instance flags, see @ref ucp_proto_common_init_flags_t */
     unsigned                flags;
+
+    /* Minimal number of iov*/
```

Comment:
space after `iov`.
minor:  I think you can add this in the middle, because this is an internal struct